Sources/TemplateKit/AST/TemplateExpression.swift

```diff
@@ -0,0 +1,39 @@
+/// An expression, like `1 + 2`, `!1`, and `a || b`
+public enum TemplateExpression {
```

Comment:
Are these enums finalized? Will they _never_ change? Or is this a breaking change waiting to happen?

Reply:
Also; is breaking these APIs permitted?

Reply:
Implementations should never be switching over these so it should be fine to add new ones. An enum that requires default for foreign modules would be nice here